tests/data/claypit/tests/phpbrowser_html_report.suite.yml

```diff
@@ -0,0 +1,9 @@
+class_name: AcceptanceTester
+modules:
+    enabled: [PhpBrowser]
+    config:
+        PhpBrowser:
+            url: https://www.google.com
```

Comment:
Initially I tried 'url: http://127.0.0.1:8000' like e.g. 'remote.suite.yml', but that failed in M$ Windows CI (see https://ci.appveyor.com/project/DavertMik/codeception/builds/27522016)